Assets/Scripts/Agent.cs

```diff
@@ -17,6 +17,12 @@ public enum FlightPhase { INITIALIZED, READY, BOOST, MIDCOURSE, TERMINAL, TERMIN
   [SerializeField]
   protected Vector3 _dragAcceleration;
 
+  
+  [SerializeField]
+  // Only for debugging (viewing in editor)
+  // Don't bother setting this it won't be used
```

Comment:
delete this comment?

Reply:
Oh it wasnt temporary, speed is really just permanently a read-only variable. Shouldn't be set. I cant make it a property (where i could do `get` with no `set`) because Unity only serialized Fields, not Properties